woowacrew-domain/src/main/java/woowacrew/github/domain/GithubCommit.java

```diff
@@ -16,6 +17,9 @@
     private LocalDate date;
     private Integer point;
 
+    private GithubCommit() {
+    }
+
     public GithubCommit(User user, LocalDate date, Integer point) {
```

Comment:
```suggestion
    public GithubCommit(User user, LocalDate date, int point) {
```

primitive로 해도 상관없지 않나요?

Reply:
넵! 피드백 반영하겠습니다 😄